catalogs/catalog-lakehouse-paimon/src/main/resources/lakehouse-paimon.conf

```diff
@@ -0,0 +1,8 @@
+#
+# Copyright 2024 Datastrato Pvt Ltd.
+# This software is licensed under the Apache License version 2.
+#
+
+## This file holds common configurations for lakehouse-paimon catalog. The format of the key is
+## 'gravitino.bypass.{paimon-inner-config-key}' and `paimon-inner-config-key` is the
+## real key that pass to lakehouse-paimon catalog.
```

Comment:
`real` to `actual`?

Reply:
@qqqttt123, `lakehouse-iceberg.conf` also uses `real`. WDYT?

Reply:
`actual` seems better